apps/blog-front/layouts/default.vue

```diff
@@ -14,6 +14,14 @@
       return { config: useConfiguration() }
     },
     head () {
+      const link = [];
+      if (this.config.themeStylesUrl) {
```

Comment:
is. there a need to guard config? 
`if (this.config?.themeStylesUrl)`

Reply:
The config is a predefined object with specific keys, it cannot be nullish.